disable-comments.php

```diff
@@ -235,7 +235,11 @@ public function filter_wp_headers( $headers ) {
 	 */
 	public function filter_query() {
 		if( is_comment_feed() ) {
-			wp_die( __( 'Comments are closed.' ), '', array( 'response' => 403 ) );
+			$refere = isset($_SERVER['HTTP_REFERER'])? $_SERVER['HTTP_REFERER']: '';
+			$location = !empty($refere)? htmlspecialchars($refere): home_url('/');
```

Comment:
I also think it is not a big problem.
Hmm, I have not know potential security errors about this kind of redirect.
If you mind, would you like to tell me about known security problem about redirect?
I'll learn more.